### PR TITLE
Add docs for sapservices gatherer

### PR DESCRIPTION
**Pull request by CDimonaco**
Add docs for sapservices gatherer.

### CODE REVIEW

File: guides/gatherers.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Here's a collection of built-in gatherers, with information about how to use the
 | [`saphostctrl@v1`](#saphostctrlv1)                                     | [trento-project/agent/../gatherers/saphostctrl.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/saphostctrl.go)                                 |
 | [`sap_profiles@v1`](#sap_profilesv1)                                   | [trento-project/agent/../gatherers/sapprofiles.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapprofiles.go)                                 |
 | [`sapinstance_hostname_resolver@v1`](#sapinstance_hostname_resolverv1) | [trento-project/agent/../gatherers/sapinstancehostnameresolver.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapinstancehostnameresolver.go) |
+| [`sapservices@v1`](#sapservicesv1)                                             | [trento-project/agent/../gatherers/sapservices.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sapservices.go)                                         |
```
Comment (arbulu89) [inline guides/gatherers.md:54]: In the raw text, I see that the table is not aligned. Fix that please

File: guides/gatherers.md [hunk 2]
```diff
@@ -1078,6 +1079,41 @@ Example output (in Rhai):
 }
 ```
 
+<span id="sapservicesv1"></span>
+
+### sapservices@v1
```
Comment (arbulu89) [inline guides/gatherers.md:1084]: Alphabetical order please....
This should come before `saptune`

**Review by arbulu89 — APPROVED** [review]
Fix alphabetical ordering and table alignment

File: guides/gatherers.md (hunk 1 above)
Comment (CDimonaco) [reply guides/gatherers.md]: done

File: guides/gatherers.md (hunk 2 above)
Comment (CDimonaco) [reply guides/gatherers.md]: done

**Pull request merged**